simphony/libraries/sipann.py

```diff
@@ -2,40 +2,690 @@
 # Licensed under the terms of the MIT License
 # (see simphony/__init__.py for details)
 
-"""
-simphony.libraries.sipann
-==============
+from simphony import Model
+from simphony.tools import freq2wl
+from SiPANN import scee, comp
```

Comment:
```suggestion
from SiPANN import scee, comp

from simphony import Model
from simphony.tools import freq2wl
```
When importing modules, Simphony imports in groups (always alphabetically):
1. Native modules
2. Dependencies
3. Simphony modules

See [simphony/formatters.py#L17-L22](https://github.com/BYUCamachoLab/simphony/blob/master/simphony/formatters.py#L17-L22) for an example